account_bank_statement_import_crelan/wizard/import_crelan_csv.py

```diff
@@ -0,0 +1,163 @@
+import csv
+import datetime
+import hashlib
+from io import StringIO
+
+from odoo import _, models
+
+ACCOUNT = "Compte donneur d'ordre"
+CURRENCY = "Devise"
+DATE = "Date"
+AMOUNT = "Montant"
+COUNTERPART_NUMBER = "Compte contrepartie"
+COUNTERPART_NAME = "Contrepartie"
+COMMUNICATION = "Communication"
+TRANSACTION_TYPE = "Type d'opération"
+
+
+class CodaBankStatementImport(models.TransientModel):
+    _inherit = "account.bank.statement.import"
```

Comment:
This entire file should be in `models/`. See https://github.com/beescoop/Obeesdoo/pull/484

Looking at this because I'm dealing with https://gestion.coopiteasy.be/web#id=9566&model=project.task&view_type=form&menu_id= as pompier.

Reply:
This model is a TransientModel. I think it nelongs to wizard. It is used to load bank files into Odoo.

```py
# odoo/addons/account_bank_statement_import

class AccountBankStatementImport(models.TransientModel):
    _name = 'account.bank.statement.import'
    _description = 'Import Bank Statement'
```

Reply:
Don't bother fixing on obeesdoo's side, I'm removing the beesdoo module anyway.